bundles/nl.esi.comma.testspecification/src/nl/esi/comma/testspecification/generator/to/fast/FromConcreteToFast.xtend

```diff
@@ -42,13 +42,18 @@ import org.eclipse.xtext.generator.IGeneratorContext
 
 import static extension nl.esi.comma.types.utilities.EcoreUtil3.*
 import java.util.LinkedHashMap
+import java.util.Arrays
 
 class FromConcreteToFast extends AbstractGenerator {
     /* TODO this should come from project task? Investigate and Implement it. */
     var record_path_for_lot_def = "ReferenceFabModelTWINSCANtooladapterandSUTTWINSCANSUTExposeInput.twinscan_expose_input.lot_definition" 
     var record_lot_def_file_name = "lot_definition"
     var record_lot_def_file_path_prefix = "./vfab2_scenario/FAST/generated_FAST/dataset/"
     
+    var record_path_for_job_def = "ReferenceFabModelYieldStartooladapterandSUTRUNYSMeasureInput.yieldstar_measure_input.job_definition" 
```

Comment:
:O this is so hacky :S we should find a way to do this properly. Didn't notice the level of hackyness till now